java-chassis-reference/en_US/general-development/file-upload.md

```diff
@@ -58,25 +58,25 @@ public String fileUpload(@RequestPart(name = "file1") MultipartFile file1, @Requ
 
 * org.springframework.core.io.Resource
 
-使用InputStream时，因为是流的方式，此时没有客户端文件名的概念，所以producer获取客户端文件名会得到null
+When using InputStream, because it is a stream, there is no concept of client file name at this time, so the producer will get the client file name will get null.
```

Comment:
so the producer will get the client file name will get null ->
so the producer gets the client file name will be null